cpp/include/raft/neighbors/detail/cagra/search_multi_kernel.cuh

```diff
@@ -404,11 +406,14 @@ __global__ void remove_parent_bit_kernel(const std::uint32_t num_queries,
                                          INDEX_T* const topk_indices_ptr,  // [ld, num_queries]
                                          const std::uint32_t ld)
 {
+  constexpr INDEX_T index_msb_1_mask = static_cast<INDEX_T>(1)
```

Comment:
Instead of repeating this definition could reuse the definition, e.g. by introducing a templated struct with a constexpr mask.

Reply:
I have modified it as you mentioned!